docs/pse-integration.mdx

```diff
@@ -141,9 +197,116 @@ Standard styling is applied to the iframe elements by default. You can override
 * `#pse-continue-pin-button` - The continue button.
 * `#pse-done-button` - The done button.
 
+### Reference CSS template for customizing PSE elements
```

Comment:
As I mentioned in many places now. Giving them an already outdated non-exhaustive list.
Web devs will know what they need to change, when they see the code.